ccl/sqlccl: TestBackupRestoreSystemJobs failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/00185f3df8fc170719a87083632e820891e47264

Parameters:
```
COCKROACH_PROPOSER_EVALUATED_KV=true
TAGS=
GOFLAGS=-race
```

Stress build found a failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=224442&tab=buildLog

```
W170416 07:48:21.543771 7314 server/status/runtime.go:111  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170416 07:48:21.567427 7314 server/config.go:403  1 storage engine initialized
I170416 07:48:21.579177 7314 server/node.go:443  [n?] store [n0,s0] not bootstrapped
I170416 07:48:21.631277 7314 server/node.go:378  [n?] **** cluster aa1fbfbf-3587-4d7d-9a60-31392a8a5fdc has been created
I170416 07:48:21.631376 7314 server/node.go:379  [n?] **** add additional nodes by specifying --join=127.0.0.1:33946
I170416 07:48:21.653272 7314 storage/store.go:1256  [n1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170416 07:48:21.653431 7314 server/node.go:456  [n1] initialized store [n1,s1]: {Capacity:536870912 Available:536870912 RangeCount:1 LeaseCount:0}
I170416 07:48:21.653586 7314 server/node.go:347  [n1] node ID 1 initialized
I170416 07:48:21.653798 7314 gossip/gossip.go:307  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:33946" > attrs:<> locality:<> 
I170416 07:48:21.657151 7314 storage/stores.go:296  [n1] read 0 node addresses from persistent storage
I170416 07:48:21.657352 7314 server/node.go:606  [n1] connecting to gossip network to verify cluster ID...
I170416 07:48:21.658331 7314 server/node.go:631  [n1] node connected via gossip and verified as part of cluster "aa1fbfbf-3587-4d7d-9a60-31392a8a5fdc"
I170416 07:48:21.658724 7314 server/node.go:394  [n1] node=1: started with [[]=] engine(s) and attributes []
I170416 07:48:21.660990 7314 sql/executor.go:326  [n1] creating distSQLPlanner with address {tcp 127.0.0.1:33946}
I170416 07:48:21.675194 7386 storage/split_queue.go:92  [split,n1,s1,r1/1:/M{in-ax},@c422194600] splitting at key /System/""
I170416 07:48:21.701213 7386 storage/replica_command.go:2620  [split,n1,s1,r1/1:/M{in-ax},@c422194600] initiating a split of this range at key /System/"" [r2]
I170416 07:48:21.705482 7314 server/server.go:706  [n1] starting https server at 127.0.0.1:33274
I170416 07:48:21.705907 7314 server/server.go:707  [n1] starting grpc/postgres server at 127.0.0.1:33946
I170416 07:48:21.706820 7314 server/server.go:708  [n1] advertising CockroachDB node at 127.0.0.1:33946
I170416 07:48:21.922102 7386 storage/split_queue.go:92  [split,n1,s1,r2/1:/{System/-Max},@c424ba6600] splitting at key /System/tsd
I170416 07:48:21.922311 7386 storage/replica_command.go:2620  [split,n1,s1,r2/1:/{System/-Max},@c424ba6600] initiating a split of this range at key /System/tsd [r3]
E170416 07:48:21.927285 7391 storage/consistency_queue.go:89  [replica consistency checker,n1,s1,r1/1:/{Min-System/},@c422194600] key range /Min-/Max outside of bounds of range /Min-/System/""
E170416 07:48:22.005709 7441 storage/replica_proposal.go:458  [n1,s1,r2/1:/{System/-Max},@c424ba6600] could not load SystemConfig span: must retry later due to intent on SystemConfigSpan
I170416 07:48:22.037151 7386 storage/split_queue.go:92  [split,n1,s1,r3/1:/{System/tsd-Max},@c42090b200] splitting at key /System/"tse"
I170416 07:48:22.037342 7386 storage/replica_command.go:2620  [split,n1,s1,r3/1:/{System/tsd-Max},@c42090b200] initiating a split of this range at key /System/"tse" [r4]
I170416 07:48:22.041069 7314 sql/event_log.go:101  [n1] Event: "alter_table", target: 12, info: {TableName:eventlog Statement:ALTER TABLE system.eventlog ALTER COLUMN uniqueID SET DEFAULT uuid_v4() User:node MutationID:0 CascadeDroppedViews:[]}
I170416 07:48:22.082887 7314 sql/lease.go:372  [n1] publish: descID=12 (eventlog) version=2 mtime=2017-04-16 07:48:22.082793999 +0000 UTC
I170416 07:48:22.184612 7386 storage/split_queue.go:92  [split,n1,s1,r4/1:/{System/tse-Max},@c42228af00] splitting at key /Table/0/0
I170416 07:48:22.184841 7386 storage/replica_command.go:2620  [split,n1,s1,r4/1:/{System/tse-Max},@c42228af00] initiating a split of this range at key /Table/0 [r5]
I170416 07:48:22.303722 7386 storage/split_queue.go:92  [split,n1,s1,r5/1:/{Table/0-Max},@c424b4b800] splitting at key /Table/11/0
I170416 07:48:22.303957 7386 storage/replica_command.go:2620  [split,n1,s1,r5/1:/{Table/0-Max},@c424b4b800] initiating a split of this range at key /Table/11 [r6]
I170416 07:48:22.353235 7386 storage/split_queue.go:92  [split,n1,s1,r6/1:/{Table/11-Max},@c42223c000] splitting at key /Table/12/0
I170416 07:48:22.353397 7386 storage/replica_command.go:2620  [split,n1,s1,r6/1:/{Table/11-Max},@c42223c000] initiating a split of this range at key /Table/12 [r7]
I170416 07:48:22.444555 7386 storage/split_queue.go:92  [split,n1,s1,r7/1:/{Table/12-Max},@c42223c600] splitting at key /Table/13/0
I170416 07:48:22.444808 7386 storage/replica_command.go:2620  [split,n1,s1,r7/1:/{Table/12-Max},@c42223c600] initiating a split of this range at key /Table/13 [r8]
I170416 07:48:22.465130 7314 server/server.go:816  [n1] done ensuring all necessary migrations have run
I170416 07:48:22.465256 7314 server/server.go:818  [n1] serving sql connections
I170416 07:48:22.548950 7717 sql/event_log.go:101  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:33946} Attrs: Locality:} ClusterID:aa1fbfbf-3587-4d7d-9a60-31392a8a5fdc StartedAt:1492328901658395679 LastUp:1492328901658395679}
I170416 07:48:22.690880 7386 storage/split_queue.go:92  [split,n1,s1,r8/1:/{Table/13-Max},@c422718300] splitting at key /Table/14/0
I170416 07:48:22.691093 7386 storage/replica_command.go:2620  [split,n1,s1,r8/1:/{Table/13-Max},@c422718300] initiating a split of this range at key /Table/14 [r9]
I170416 07:48:22.840931 7386 storage/split_queue.go:92  [split,n1,s1,r9/1:/{Table/14-Max},@c422797b00] splitting at key /Table/15/0
I170416 07:48:22.841121 7386 storage/replica_command.go:2620  [split,n1,s1,r9/1:/{Table/14-Max},@c422797b00] initiating a split of this range at key /Table/15 [r10]
W170416 07:48:23.090558 7314 gossip/gossip.go:1192  [n?] no incoming or outgoing connections
W170416 07:48:23.232927 7314 server/status/runtime.go:111  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170416 07:48:23.246742 7314 server/config.go:403  1 storage engine initialized
I170416 07:48:23.302528 7314 server/node.go:443  [n?] store [n0,s0] not bootstrapped
I170416 07:48:23.302674 7314 storage/stores.go:296  [n?] read 0 node addresses from persistent storage
I170416 07:48:23.302790 7314 server/node.go:606  [n?] connecting to gossip network to verify cluster ID...
I170416 07:48:23.459288 7920 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:33946
I170416 07:48:23.460989 7927 gossip/server.go:234  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:39522}
I170416 07:48:23.470491 7314 server/node.go:631  [n?] node connected via gossip and verified as part of cluster "aa1fbfbf-3587-4d7d-9a60-31392a8a5fdc"
I170416 07:48:23.470923 7850 storage/stores.go:312  [n?] wrote 1 node addresses to persistent storage
I170416 07:48:23.494231 7314 kv/dist_sender.go:360  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170416 07:48:23.500145 7314 server/node.go:340  [n?] new node allocated ID 2
I170416 07:48:23.500487 7314 gossip/gossip.go:307  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:39522" > attrs:<> locality:<> 
I170416 07:48:23.501247 7314 server/node.go:394  [n2] node=2: started with [[]=] engine(s) and attributes []
I170416 07:48:23.503729 7314 sql/executor.go:326  [n2] creating distSQLPlanner with address {tcp 127.0.0.1:39522}
I170416 07:48:23.514487 7956 storage/stores.go:312  [n1] wrote 1 node addresses to persistent storage
I170416 07:48:23.607118 7314 server/server.go:706  [n2] starting https server at 127.0.0.1:42702
I170416 07:48:23.607299 7314 server/server.go:707  [n2] starting grpc/postgres server at 127.0.0.1:39522
I170416 07:48:23.610443 7314 server/server.go:708  [n2] advertising CockroachDB node at 127.0.0.1:39522
I170416 07:48:23.624722 7314 server/server.go:816  [n2] done ensuring all necessary migrations have run
I170416 07:48:23.624868 7314 server/server.go:818  [n2] serving sql connections
I170416 07:48:23.624998 7932 server/node.go:587  [n2] bootstrapped store [n2,s2]
I170416 07:48:23.632076 7290 storage/replica_raftstorage.go:413  [replicate,n1,s1,r7/1:/Table/1{2-3},@c42223c600] generated preemptive snapshot 1fe95f58 at index 22
I170416 07:48:23.714753 8089 sql/event_log.go:101  [n2] Event: "node_join", target: 2, info: {Descriptor:{NodeID:2 Address:{NetworkField:tcp AddressField:127.0.0.1:39522} Attrs: Locality:} ClusterID:aa1fbfbf-3587-4d7d-9a60-31392a8a5fdc StartedAt:1492328903501042843 LastUp:1492328903501042843}
W170416 07:48:23.827559 7314 gossip/gossip.go:1192  [n?] no incoming or outgoing connections
W170416 07:48:23.848233 7314 server/status/runtime.go:111  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I170416 07:48:23.897671 7314 server/config.go:403  1 storage engine initialized
I170416 07:48:23.899049 7314 server/node.go:443  [n?] store [n0,s0] not bootstrapped
I170416 07:48:23.899202 7314 storage/stores.go:296  [n?] read 0 node addresses from persistent storage
I170416 07:48:23.899315 7314 server/node.go:606  [n?] connecting to gossip network to verify cluster ID...
I170416 07:48:23.904972 7952 storage/replica_raftstorage.go:593  [n2,s2,r7/?:{-},@c423a02600] applying preemptive snapshot at index 22 (id=1fe95f58, encoded size=12584, 1 rocksdb batches, 12 log entries)
I170416 07:48:23.908914 7952 storage/replica_raftstorage.go:601  [n2,s2,r7/?:/Table/1{2-3},@c423a02600] applied preemptive snapshot in 4ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170416 07:48:23.915345 7290 storage/store.go:3320  [replicate,n1,s1,r7/1:/Table/1{2-3},@c42223c600] streamed snapshot: kv pairs: 21, log entries: 12, 50ms
I170416 07:48:23.925283 7290 storage/replica_command.go:3515  [replicate,n1,s1,r7/1:/Table/1{2-3},@c42223c600] change replicas (ADD_REPLICA {2 2 2}): read existing descriptor r7:/Table/1{2-3} [r1(n1,s1), next=2]
I170416 07:48:23.958565 8174 storage/replica.go:2637  [n1,s1,r7/1:/Table/1{2-3},@c42223c600] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170416 07:48:23.962424 7387 storage/replica_raftstorage.go:413  [replicate,n1,s1,r6/1:/Table/1{1-2},@c42223c000] generated preemptive snapshot 0edde2be at index 18
I170416 07:48:23.965218 7387 storage/store.go:3320  [replicate,n1,s1,r6/1:/Table/1{1-2},@c42223c000] streamed snapshot: kv pairs: 12, log entries: 8, 2ms
I170416 07:48:23.973050 8177 storage/replica_raftstorage.go:593  [n2,s2,r6/?:{-},@c4248f6300] applying preemptive snapshot at index 18 (id=0edde2be, encoded size=6465, 1 rocksdb batches, 8 log entries)
I170416 07:48:23.974051 8141 storage/raft_transport.go:436  [n2] raft transport stream to node 1 established
I170416 07:48:23.974526 8177 storage/replica_raftstorage.go:601  [n2,s2,r6/?:/Table/1{1-2},@c4248f6300] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170416 07:48:23.990767 7387 storage/replica_command.go:3515  [replicate,n1,s1,r6/1:/Table/1{1-2},@c42223c000] change replicas (ADD_REPLICA {2 2 2}): read existing descriptor r6:/Table/1{1-2} [r1(n1,s1), next=2]
I170416 07:48:24.040842 8104 gossip/client.go:131  [n?] started gossip client to 127.0.0.1:33946
I170416 07:48:24.045503 8145 gossip/server.go:234  [n1] received initial cluster-verification connection from {tcp 127.0.0.1:60989}
I170416 07:48:24.053795 8197 storage/stores.go:312  [n?] wrote 1 node addresses to persistent storage
I170416 07:48:24.054659 8197 storage/stores.go:312  [n?] wrote 2 node addresses to persistent storage
I170416 07:48:24.056762 7314 server/node.go:631  [n?] node connected via gossip and verified as part of cluster "aa1fbfbf-3587-4d7d-9a60-31392a8a5fdc"
I170416 07:48:24.084519 7314 kv/dist_sender.go:360  [n?] unable to determine this node's attributes for replica selection; node is most likely bootstrapping
I170416 07:48:24.089922 7314 server/node.go:340  [n?] new node allocated ID 3
I170416 07:48:24.090218 7314 gossip/gossip.go:307  [n3] NodeDescriptor set to node_id:3 address:<network_field:"tcp" address_field:"127.0.0.1:60989" > attrs:<> locality:<> 
I170416 07:48:24.091046 7314 server/node.go:394  [n3] node=3: started with [[]=] engine(s) and attributes []
I170416 07:48:24.094474 7871 storage/stores.go:312  [n1] wrote 2 node addresses to persistent storage
I170416 07:48:24.099137 7872 storage/stores.go:312  [n2] wrote 2 node addresses to persistent storage
I170416 07:48:24.110304 7314 sql/executor.go:326  [n3] creating distSQLPlanner with address {tcp 127.0.0.1:60989}
I170416 07:48:24.113327 8228 storage/replica.go:2637  [n1,s1,r6/1:/Table/1{1-2},@c42223c000] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170416 07:48:24.117693 7290 storage/replica_raftstorage.go:413  [replicate,n1,s1,r9/1:/Table/1{4-5},@c422797b00] generated preemptive snapshot 11908b14 at index 18
I170416 07:48:24.120461 7290 storage/store.go:3320  [replicate,n1,s1,r9/1:/Table/1{4-5},@c422797b00] streamed snapshot: kv pairs: 11, log entries: 8, 2ms
I170416 07:48:24.121739 8388 storage/replica_raftstorage.go:593  [n2,s2,r9/?:{-},@c4248f6f00] applying preemptive snapshot at index 18 (id=11908b14, encoded size=5584, 1 rocksdb batches, 8 log entries)
I170416 07:48:24.123014 8388 storage/replica_raftstorage.go:601  [n2,s2,r9/?:/Table/1{4-5},@c4248f6f00] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170416 07:48:24.142935 7290 storage/replica_command.go:3515  [replicate,n1,s1,r9/1:/Table/1{4-5},@c422797b00] change replicas (ADD_REPLICA {2 2 2}): read existing descriptor r9:/Table/1{4-5} [r1(n1,s1), next=2]
I170416 07:48:24.182057 8159 server/node.go:587  [n3] bootstrapped store [n3,s3]
I170416 07:48:24.213094 7314 server/server.go:706  [n3] starting https server at 127.0.0.1:51886
I170416 07:48:24.215094 7314 server/server.go:707  [n3] starting grpc/postgres server at 127.0.0.1:60989
I170416 07:48:24.215161 7314 server/server.go:708  [n3] advertising CockroachDB node at 127.0.0.1:60989
I170416 07:48:24.288734 7314 server/server.go:816  [n3] done ensuring all necessary migrations have run
I170416 07:48:24.288858 7314 server/server.go:818  [n3] serving sql connections
I170416 07:48:24.338267 8362 storage/replica.go:2637  [n1,s1,r9/1:/Table/1{4-5},@c422797b00] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170416 07:48:24.341658 7290 storage/replica_raftstorage.go:413  [replicate,n1,s1,r6/1:/Table/1{1-2},@c42223c000] generated preemptive snapshot 9578054a at index 21
I170416 07:48:24.456180 8415 sql/event_log.go:101  [n3] Event: "node_join", target: 3, info: {Descriptor:{NodeID:3 Address:{NetworkField:tcp AddressField:127.0.0.1:60989} Attrs: Locality:} ClusterID:aa1fbfbf-3587-4d7d-9a60-31392a8a5fdc StartedAt:1492328904090803841 LastUp:1492328904090803841}
I170416 07:48:24.551921 8238 storage/replica_raftstorage.go:593  [n3,s3,r6/?:{-},@c422796600] applying preemptive snapshot at index 21 (id=9578054a, encoded size=8538, 1 rocksdb batches, 11 log entries)
I170416 07:48:24.554466 8238 storage/replica_raftstorage.go:601  [n3,s3,r6/?:/Table/1{1-2},@c422796600] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170416 07:48:24.560534 7290 storage/store.go:3320  [replicate,n1,s1,r6/1:/Table/1{1-2},@c42223c000] streamed snapshot: kv pairs: 13, log entries: 11, 16ms
I170416 07:48:24.562602 7290 storage/replica_command.go:3515  [replicate,n1,s1,r6/1:/Table/1{1-2},@c42223c000] change replicas (ADD_REPLICA {3 3 3}): read existing descriptor r6:/Table/1{1-2} [r1(n1,s1), r2(n2,s2), next=3]
I170416 07:48:24.598751 8433 storage/replica.go:2637  [n1,s1,r6/1:/Table/1{1-2},@c42223c000] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170416 07:48:24.615155 8516 storage/raft_transport.go:436  [n3] raft transport stream to node 1 established
I170416 07:48:24.622181 7290 storage/replica_raftstorage.go:413  [replicate,n1,s1,r1/1:/{Min-System/},@c422194600] generated preemptive snapshot 8fe180b5 at index 58
I170416 07:48:24.635560 8494 storage/replica_raftstorage.go:593  [n3,s3,r1/?:{-},@c422760c00] applying preemptive snapshot at index 58 (id=8fe180b5, encoded size=80954, 1 rocksdb batches, 48 log entries)
I170416 07:48:24.636191 7290 storage/store.go:3320  [replicate,n1,s1,r1/1:/{Min-System/},@c422194600] streamed snapshot: kv pairs: 37, log entries: 48, 11ms
I170416 07:48:24.640481 8494 storage/replica_raftstorage.go:601  [n3,s3,r1/?:/{Min-System/},@c422760c00] applied preemptive snapshot in 5ms [clear=0ms batch=0ms entries=2ms commit=1ms]
I170416 07:48:24.643505 7290 storage/replica_command.go:3515  [replicate,n1,s1,r1/1:/{Min-System/},@c422194600] change replicas (ADD_REPLICA {3 3 2}): read existing descriptor r1:/{Min-System/} [r1(n1,s1), next=2]
I170416 07:48:24.682288 8505 storage/replica.go:2637  [n1,s1,r1/1:/{Min-System/},@c422194600] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170416 07:48:24.686512 7290 storage/replica_raftstorage.go:413  [replicate,n1,s1,r3/1:/System/ts{d-e},@c42090b200] generated preemptive snapshot eee71451 at index 28
I170416 07:48:24.720802 7290 storage/store.go:3320  [replicate,n1,s1,r3/1:/System/ts{d-e},@c42090b200] streamed snapshot: kv pairs: 801, log entries: 18, 26ms
I170416 07:48:24.722007 8453 storage/replica_raftstorage.go:593  [n3,s3,r3/?:{-},@c422719500] applying preemptive snapshot at index 28 (id=eee71451, encoded size=142604, 1 rocksdb batches, 18 log entries)
I170416 07:48:24.724861 8453 storage/replica_raftstorage.go:601  [n3,s3,r3/?:/System/ts{d-e},@c422719500] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=1ms commit=1ms]
I170416 07:48:24.738281 7290 storage/replica_command.go:3515  [replicate,n1,s1,r3/1:/System/ts{d-e},@c42090b200] change replicas (ADD_REPLICA {3 3 2}): read existing descriptor r3:/System/ts{d-e} [r1(n1,s1), next=2]
I170416 07:48:24.769655 8541 storage/replica.go:2637  [n1,s1,r3/1:/System/ts{d-e},@c42090b200] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170416 07:48:24.796323 7290 storage/replica_raftstorage.go:413  [replicate,n1,s1,r5/1:/Table/{0-11},@c424b4b800] generated preemptive snapshot e8ef3e2f at index 22
I170416 07:48:24.801295 7290 storage/store.go:3320  [replicate,n1,s1,r5/1:/Table/{0-11},@c424b4b800] streamed snapshot: kv pairs: 40, log entries: 12, 4ms
I170416 07:48:24.803875 8579 storage/replica_raftstorage.go:593  [n2,s2,r5/?:{-},@c422715500] applying preemptive snapshot at index 22 (id=e8ef3e2f, encoded size=15498, 1 rocksdb batches, 12 log entries)
I170416 07:48:24.805281 8579 storage/replica_raftstorage.go:601  [n2,s2,r5/?:/Table/{0-11},@c422715500] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170416 07:48:24.809636 7290 storage/replica_command.go:3515  [replicate,n1,s1,r5/1:/Table/{0-11},@c424b4b800] change replicas (ADD_REPLICA {2 2 2}): read existing descriptor r5:/Table/{0-11} [r1(n1,s1), next=2]
I170416 07:48:24.855454 8600 storage/replica.go:2637  [n1,s1,r5/1:/Table/{0-11},@c424b4b800] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170416 07:48:24.863077 7290 storage/replica_raftstorage.go:413  [replicate,n1,s1,r10/1:/{Table/15-Max},@c42229af00] generated preemptive snapshot 0c8c5b1d at index 11
I170416 07:48:24.868541 7290 storage/store.go:3320  [replicate,n1,s1,r10/1:/{Table/15-Max},@c42229af00] streamed snapshot: kv pairs: 10, log entries: 1, 5ms
I170416 07:48:24.873115 8456 storage/replica_raftstorage.go:593  [n3,s3,r10/?:{-},@c42090a600] applying preemptive snapshot at index 11 (id=0c8c5b1d, encoded size=580, 1 rocksdb batches, 1 log entries)
I170416 07:48:24.874190 8456 storage/replica_raftstorage.go:601  [n3,s3,r10/?:/{Table/15-Max},@c42090a600] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170416 07:48:24.884598 7290 storage/replica_command.go:3515  [replicate,n1,s1,r10/1:/{Table/15-Max},@c42229af00] change replicas (ADD_REPLICA {3 3 2}): read existing descriptor r10:/{Table/15-Max} [r1(n1,s1), next=2]
I170416 07:48:24.971655 8526 storage/replica.go:2637  [n1,s1,r10/1:/{Table/15-Max},@c42229af00] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170416 07:48:24.981214 7290 storage/replica_raftstorage.go:413  [replicate,n1,s1,r8/1:/Table/1{3-4},@c422718300] generated preemptive snapshot 8c168ca0 at index 40
I170416 07:48:25.002056 8584 storage/replica_raftstorage.go:593  [n2,s2,r8/?:{-},@c42273e300] applying preemptive snapshot at index 40 (id=8c168ca0, encoded size=38724, 1 rocksdb batches, 30 log entries)
I170416 07:48:25.004009 8584 storage/replica_raftstorage.go:601  [n2,s2,r8/?:/Table/1{3-4},@c42273e300] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170416 07:48:25.009156 7290 storage/store.go:3320  [replicate,n1,s1,r8/1:/Table/1{3-4},@c422718300] streamed snapshot: kv pairs: 111, log entries: 30, 21ms
I170416 07:48:25.034726 7290 storage/replica_command.go:3515  [replicate,n1,s1,r8/1:/Table/1{3-4},@c422718300] change replicas (ADD_REPLICA {2 2 2}): read existing descriptor r8:/Table/1{3-4} [r1(n1,s1), next=2]
I170416 07:48:25.083801 8560 storage/replica.go:2637  [n1,s1,r8/1:/Table/1{3-4},@c422718300] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2}]
I170416 07:48:25.089329 7387 storage/replica_raftstorage.go:413  [replicate,n1,s1,r1/1:/{Min-System/},@c422194600] generated preemptive snapshot 056ead27 at index 68
I170416 07:48:25.096611 7387 storage/store.go:3320  [replicate,n1,s1,r1/1:/{Min-System/},@c422194600] streamed snapshot: kv pairs: 45, log entries: 58, 6ms
I170416 07:48:25.098957 8630 storage/replica_raftstorage.go:593  [n2,s2,r1/?:{-},@c422815800] applying preemptive snapshot at index 68 (id=056ead27, encoded size=86978, 1 rocksdb batches, 58 log entries)
I170416 07:48:25.103590 8630 storage/replica_raftstorage.go:601  [n2,s2,r1/?:/{Min-System/},@c422815800] applied preemptive snapshot in 4ms [clear=0ms batch=0ms entries=3ms commit=0ms]
I170416 07:48:25.108245 7387 storage/replica_command.go:3515  [replicate,n1,s1,r1/1:/{Min-System/},@c422194600] change replicas (ADD_REPLICA {2 2 3}): read existing descriptor r1:/{Min-System/} [r1(n1,s1), r2(n3,s3), next=3]
I170416 07:48:25.146804 8659 storage/replica.go:2637  [n1,s1,r1/1:/{Min-System/},@c422194600] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170416 07:48:25.174936 7290 storage/replica_raftstorage.go:413  [replicate,n1,s1,r2/1:/System/{-tsd},@c424ba6600] generated preemptive snapshot 1e73ef35 at index 37
I170416 07:48:25.184355 7290 storage/store.go:3320  [replicate,n1,s1,r2/1:/System/{-tsd},@c424ba6600] streamed snapshot: kv pairs: 32, log entries: 27, 5ms
I170416 07:48:25.186831 8240 storage/replica_raftstorage.go:593  [n3,s3,r2/?:{-},@c421c6a600] applying preemptive snapshot at index 37 (id=1e73ef35, encoded size=152346, 1 rocksdb batches, 27 log entries)
I170416 07:48:25.190216 8240 storage/replica_raftstorage.go:601  [n3,s3,r2/?:/System/{-tsd},@c421c6a600] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170416 07:48:25.223107 7290 storage/replica_command.go:3515  [replicate,n1,s1,r2/1:/System/{-tsd},@c424ba6600] change replicas (ADD_REPLICA {3 3 2}): read existing descriptor r2:/System/{-tsd} [r1(n1,s1), next=2]
I170416 07:48:25.321367 8661 storage/replica.go:2637  [n1,s1,r2/1:/System/{-tsd},@c424ba6600] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170416 07:48:25.338578 7290 storage/replica_raftstorage.go:413  [replicate,n1,s1,r4/1:/{System/tse-Table/0},@c42228af00] generated preemptive snapshot f659cd81 at index 21
I170416 07:48:25.342962 7290 storage/store.go:3320  [replicate,n1,s1,r4/1:/{System/tse-Table/0},@c42228af00] streamed snapshot: kv pairs: 13, log entries: 11, 4ms
I170416 07:48:25.344976 8713 storage/replica_raftstorage.go:593  [n3,s3,r4/?:{-},@c421eb3b00] applying preemptive snapshot at index 21 (id=f659cd81, encoded size=8677, 1 rocksdb batches, 11 log entries)
I170416 07:48:25.346432 8713 storage/replica_raftstorage.go:601  [n3,s3,r4/?:/{System/tse-Table/0},@c421eb3b00] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170416 07:48:25.363539 7290 storage/replica_command.go:3515  [replicate,n1,s1,r4/1:/{System/tse-Table/0},@c42228af00] change replicas (ADD_REPLICA {3 3 2}): read existing descriptor r4:/{System/tse-Table/0} [r1(n1,s1), next=2]
I170416 07:48:25.412491 8605 storage/replica.go:2637  [n1,s1,r4/1:/{System/tse-Table/0},@c42228af00] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:2}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2}]
I170416 07:48:25.415693 7290 storage/queue.go:699  [n1,replicate] purgatory is now empty
I170416 07:48:25.416763 7387 storage/replica_raftstorage.go:413  [replicate,n1,s1,r5/1:/Table/{0-11},@c424b4b800] generated preemptive snapshot 7f3d47b8 at index 25
I170416 07:48:25.420708 7387 storage/store.go:3320  [replicate,n1,s1,r5/1:/Table/{0-11},@c424b4b800] streamed snapshot: kv pairs: 41, log entries: 15, 3ms
I170416 07:48:25.427519 8694 storage/replica_raftstorage.go:593  [n3,s3,r5/?:{-},@c421972300] applying preemptive snapshot at index 25 (id=7f3d47b8, encoded size=17571, 1 rocksdb batches, 15 log entries)
I170416 07:48:25.429185 8694 storage/replica_raftstorage.go:601  [n3,s3,r5/?:/Table/{0-11},@c421972300] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=1ms commit=0ms]
W170416 07:48:25.452862 7420 vendor/github.com/coreos/etcd/raft/raft.go:793  [n1,s1,r4/1:/{System/tse-Table/0},@c42228af00] 1 stepped down to follower since quorum is not active
I170416 07:48:25.453067 7387 storage/replica_command.go:3515  [replicate,n1,s1,r5/1:/Table/{0-11},@c424b4b800] change replicas (ADD_REPLICA {3 3 3}): read existing descriptor r5:/Table/{0-11} [r1(n1,s1), r2(n2,s2), next=3]
I170416 07:48:25.495663 8741 storage/replica.go:2637  [n1,s1,r5/1:/Table/{0-11},@c424b4b800] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170416 07:48:25.517824 7387 storage/replica_raftstorage.go:413  [replicate,n1,s1,r10/1:/{Table/15-Max},@c42229af00] generated preemptive snapshot d04d1879 at index 16
I170416 07:48:25.541970 7387 storage/store.go:3320  [replicate,n1,s1,r10/1:/{Table/15-Max},@c42229af00] streamed snapshot: kv pairs: 12, log entries: 6, 23ms
I170416 07:48:25.545982 8717 storage/replica_raftstorage.go:593  [n2,s2,r10/?:{-},@c420da6300] applying preemptive snapshot at index 16 (id=d04d1879, encoded size=3616, 1 rocksdb batches, 6 log entries)
I170416 07:48:25.547291 8717 storage/replica_raftstorage.go:601  [n2,s2,r10/?:/{Table/15-Max},@c420da6300] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170416 07:48:25.557223 7387 storage/replica_command.go:3515  [replicate,n1,s1,r10/1:/{Table/15-Max},@c42229af00] change replicas (ADD_REPLICA {2 2 3}): read existing descriptor r10:/{Table/15-Max} [r1(n1,s1), r2(n3,s3), next=3]
I170416 07:48:25.593663 8721 storage/replica.go:2637  [n1,s1,r10/1:/{Table/15-Max},@c42229af00] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170416 07:48:25.616559 7387 storage/replica_raftstorage.go:413  [replicate,n1,s1,r4/1:/{System/tse-Table/0},@c42228af00] generated preemptive snapshot 1c3801f4 at index 23
I170416 07:48:25.629377 7387 storage/store.go:3320  [replicate,n1,s1,r4/1:/{System/tse-Table/0},@c42228af00] streamed snapshot: kv pairs: 15, log entries: 13, 12ms
I170416 07:48:25.636792 8786 storage/replica_raftstorage.go:593  [n2,s2,r4/?:{-},@c421dbc900] applying preemptive snapshot at index 23 (id=1c3801f4, encoded size=10638, 1 rocksdb batches, 13 log entries)
I170416 07:48:25.664341 8786 storage/replica_raftstorage.go:601  [n2,s2,r4/?:/{System/tse-Table/0},@c421dbc900] applied preemptive snapshot in 27ms [clear=0ms batch=0ms entries=0ms commit=1ms]
I170416 07:48:25.669537 7387 storage/replica_command.go:3515  [replicate,n1,s1,r4/1:/{System/tse-Table/0},@c42228af00] change replicas (ADD_REPLICA {2 2 3}): read existing descriptor r4:/{System/tse-Table/0} [r1(n1,s1), r2(n3,s3), next=3]
I170416 07:48:29.718556 8780 storage/replica.go:2637  [n1,s1,r4/1:/{System/tse-Table/0},@c42228af00] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170416 07:48:29.753721 7387 storage/replica_raftstorage.go:413  [replicate,n1,s1,r2/1:/System/{-tsd},@c424ba6600] generated preemptive snapshot 2dc2899a at index 44
I170416 07:48:29.772820 7387 storage/store.go:3320  [replicate,n1,s1,r2/1:/System/{-tsd},@c424ba6600] streamed snapshot: kv pairs: 36, log entries: 34, 18ms
I170416 07:48:29.776784 8850 storage/replica_raftstorage.go:593  [n2,s2,r2/?:{-},@c421a1c300] applying preemptive snapshot at index 44 (id=2dc2899a, encoded size=156491, 1 rocksdb batches, 34 log entries)
I170416 07:48:29.781533 8850 storage/replica_raftstorage.go:601  [n2,s2,r2/?:/System/{-tsd},@c421a1c300] applied preemptive snapshot in 5ms [clear=0ms batch=0ms entries=3ms commit=1ms]
I170416 07:48:29.784374 7387 storage/replica_command.go:3515  [replicate,n1,s1,r2/1:/System/{-tsd},@c424ba6600] change replicas (ADD_REPLICA {2 2 3}): read existing descriptor r2:/System/{-tsd} [r1(n1,s1), r2(n3,s3), next=3]
I170416 07:48:29.815845 8783 storage/replica.go:2637  [n1,s1,r2/1:/System/{-tsd},@c424ba6600] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170416 07:48:29.826515 7387 storage/replica_raftstorage.go:413  [replicate,n1,s1,r3/1:/System/ts{d-e},@c42090b200] generated preemptive snapshot 3db17f45 at index 31
I170416 07:48:29.874754 7387 storage/store.go:3320  [replicate,n1,s1,r3/1:/System/ts{d-e},@c42090b200] streamed snapshot: kv pairs: 802, log entries: 21, 48ms
I170416 07:48:29.881043 8854 storage/replica_raftstorage.go:593  [n2,s2,r3/?:{-},@c4223bbb00] applying preemptive snapshot at index 31 (id=3db17f45, encoded size=144746, 1 rocksdb batches, 21 log entries)
I170416 07:48:29.884528 8854 storage/replica_raftstorage.go:601  [n2,s2,r3/?:/System/ts{d-e},@c4223bbb00] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=1ms commit=1ms]
I170416 07:48:29.894556 7387 storage/replica_command.go:3515  [replicate,n1,s1,r3/1:/System/ts{d-e},@c42090b200] change replicas (ADD_REPLICA {2 2 3}): read existing descriptor r3:/System/ts{d-e} [r1(n1,s1), r2(n3,s3), next=3]
I170416 07:48:29.986149 8829 storage/replica.go:2637  [n1,s1,r3/1:/System/ts{d-e},@c42090b200] proposing ADD_REPLICA {NodeID:2 StoreID:2 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:3 StoreID:3 ReplicaID:2} {NodeID:2 StoreID:2 ReplicaID:3}]
I170416 07:48:30.009955 7387 storage/replica_raftstorage.go:413  [replicate,n1,s1,r7/1:/Table/1{2-3},@c42223c600] generated preemptive snapshot 8083465b at index 31
I170416 07:48:30.077774 7387 storage/store.go:3320  [replicate,n1,s1,r7/1:/Table/1{2-3},@c42223c600] streamed snapshot: kv pairs: 33, log entries: 21, 24ms
I170416 07:48:30.079473 8830 storage/replica_raftstorage.go:593  [n3,s3,r7/?:{-},@c421ba5200] applying preemptive snapshot at index 31 (id=8083465b, encoded size=21882, 1 rocksdb batches, 21 log entries)
I170416 07:48:30.081372 8830 storage/replica_raftstorage.go:601  [n3,s3,r7/?:/Table/1{2-3},@c421ba5200] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170416 07:48:30.087040 7387 storage/replica_command.go:3515  [replicate,n1,s1,r7/1:/Table/1{2-3},@c42223c600] change replicas (ADD_REPLICA {3 3 3}): read existing descriptor r7:/Table/1{2-3} [r1(n1,s1), r2(n2,s2), next=3]
I170416 07:48:30.132827 8789 storage/replica.go:2637  [n1,s1,r7/1:/Table/1{2-3},@c42223c600] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170416 07:48:30.144236 7387 storage/replica_raftstorage.go:413  [replicate,n1,s1,r9/1:/Table/1{4-5},@c422797b00] generated preemptive snapshot 297e959e at index 23
I170416 07:48:30.180296 7387 storage/store.go:3320  [replicate,n1,s1,r9/1:/Table/1{4-5},@c422797b00] streamed snapshot: kv pairs: 13, log entries: 13, 35ms
I170416 07:48:30.182071 8815 storage/replica_raftstorage.go:593  [n3,s3,r9/?:{-},@c42087c000] applying preemptive snapshot at index 23 (id=297e959e, encoded size=8614, 1 rocksdb batches, 13 log entries)
I170416 07:48:30.183430 8815 storage/replica_raftstorage.go:601  [n3,s3,r9/?:/Table/1{4-5},@c42087c000] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170416 07:48:30.193029 7387 storage/replica_command.go:3515  [replicate,n1,s1,r9/1:/Table/1{4-5},@c422797b00] change replicas (ADD_REPLICA {3 3 3}): read existing descriptor r9:/Table/1{4-5} [r1(n1,s1), r2(n2,s2), next=3]
I170416 07:48:30.220485 8914 storage/replica.go:2637  [n1,s1,r9/1:/Table/1{4-5},@c422797b00] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170416 07:48:30.227700 7387 storage/replica_raftstorage.go:413  [replicate,n1,s1,r8/1:/Table/1{3-4},@c422718300] generated preemptive snapshot d46d7590 at index 64
I170416 07:48:30.257864 7387 storage/store.go:3320  [replicate,n1,s1,r8/1:/Table/1{3-4},@c422718300] streamed snapshot: kv pairs: 167, log entries: 54, 29ms
I170416 07:48:30.261891 8911 storage/replica_raftstorage.go:593  [n3,s3,r8/?:{-},@c4207f3200] applying preemptive snapshot at index 64 (id=d46d7590, encoded size=67824, 1 rocksdb batches, 54 log entries)
I170416 07:48:30.268064 8911 storage/replica_raftstorage.go:601  [n3,s3,r8/?:/Table/1{3-4},@c4207f3200] applied preemptive snapshot in 6ms [clear=0ms batch=0ms entries=5ms commit=0ms]
I170416 07:48:30.276187 7387 storage/replica_command.go:3515  [replicate,n1,s1,r8/1:/Table/1{3-4},@c422718300] change replicas (ADD_REPLICA {3 3 3}): read existing descriptor r8:/Table/1{3-4} [r1(n1,s1), r2(n2,s2), next=3]
I170416 07:48:30.309424 8921 storage/replica.go:2637  [n1,s1,r8/1:/Table/1{3-4},@c422718300] proposing ADD_REPLICA {NodeID:3 StoreID:3 ReplicaID:3}: [{NodeID:1 StoreID:1 ReplicaID:1} {NodeID:2 StoreID:2 ReplicaID:2} {NodeID:3 StoreID:3 ReplicaID:3}]
I170416 07:48:51.417807 7919 gossip/gossip.go:1206  [n2] node has connected to cluster via gossip
I170416 07:48:51.418416 7919 storage/stores.go:312  [n2] wrote 2 node addresses to persistent storage
I170416 07:48:52.501454 7330 gossip/gossip.go:1206  [n1] node has connected to cluster via gossip
I170416 07:48:52.502050 7330 storage/stores.go:312  [n1] wrote 2 node addresses to persistent storage
I170416 07:49:00.187510 8172 gossip/gossip.go:1206  [n3] node has connected to cluster via gossip
I170416 07:49:00.188072 8172 storage/stores.go:312  [n3] wrote 2 node addresses to persistent storage
	<autogenerated>:14: ccl/sqlccl/backup_test.go:121, condition failed to evaluate within 45s: enterprise.enabled is not true
```

The last couple failures all have a common pattern:

```
[07:00:09][Step 2/2] I170602 06:56:51.247857 16512 util/stop/stopper.go:505  quiescing; tasks left:
[07:00:09][Step 2/2] 8      server/node.go:855
[07:00:09][Step 2/2] 1      storage/queue.go:523
[07:00:09][Step 2/2] I170602 06:56:51.250327 16512 util/stop/stopper.go:505  quiescing; tasks left:
[07:00:09][Step 2/2] 7      server/node.go:855
[07:00:09][Step 2/2] 1      storage/queue.go:523
[07:00:09][Step 2/2] I170602 06:56:51.251005 16512 util/stop/stopper.go:505  quiescing; tasks left:
[07:00:09][Step 2/2] 6      server/node.go:855
[07:00:09][Step 2/2] 1      storage/queue.go:523
[07:00:09][Step 2/2] I170602 06:56:51.251649 16512 util/stop/stopper.go:505  quiescing; tasks left:
[07:00:09][Step 2/2] 5      server/node.go:855
[07:00:09][Step 2/2] 1      storage/queue.go:523
[07:00:09][Step 2/2] E170602 06:56:51.256065 16184 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): node unavailable; try another peer
[07:00:09][Step 2/2] E170602 06:56:51.257664 16188 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): node unavailable; try another peer
[07:00:09][Step 2/2] W170602 06:56:51.258309 15380 storage/raft_transport.go:442  [n2] raft transport stream to node 3 failed: EOF
[07:00:09][Step 2/2] E170602 06:56:51.259059 16181 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): node unavailable; try another peer
[07:00:09][Step 2/2] E170602 06:56:51.259590 16187 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): node unavailable; try another peer
[07:00:09][Step 2/2] W170602 06:56:51.260823 14926 storage/raft_transport.go:442  [n3] raft transport stream to node 1 failed: rpc error: code = Internal desc = transport is closing
[07:00:09][Step 2/2] E170602 06:56:51.261223 16185 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): node unavailable; try another peer
[07:00:09][Step 2/2] W170602 06:56:51.261354 15350 storage/raft_transport.go:442  [n3] raft transport stream to node 2 failed: rpc error: code = Internal desc = transport is closing
[07:00:09][Step 2/2] E170602 06:56:51.263402 16183 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): node unavailable; try another peer
[07:00:09][Step 2/2] E170602 06:56:51.263571 16186 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): node unavailable; try another peer
[07:00:09][Step 2/2] E170602 06:56:51.264627 16180 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): node unavailable; try another peer
[07:00:09][Step 2/2] E170602 06:56:51.264900 16182 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): node unavailable; try another peer
[07:00:09][Step 2/2] I170602 06:56:51.268416 13952 vendor/google.golang.org/grpc/transport/http2_client.go:1283  transport: http2Client.notifyError got notified that the client transport was broken EOF.
[07:00:09][Step 2/2] I170602 06:56:51.269516 13708 vendor/google.golang.org/grpc/clientconn.go:855  grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:58952: operation was canceled"; Reconnecting to {127.0.0.1:58952 <nil>}
[07:00:09][Step 2/2] I170602 06:56:51.269671 13708 vendor/google.golang.org/grpc/clientconn.go:957  grpc: addrConn.transportMonitor exits due to: grpc: the connection is closing
[07:00:09][Step 2/2] W170602 06:56:51.296320 14287 storage/raft_transport.go:442  [n2] raft transport stream to node 1 failed: rpc error: code = Internal desc = transport is closing
[07:00:09][Step 2/2] W170602 06:56:51.296585 14880 storage/raft_transport.go:442  [n1] raft transport stream to node 3 failed: EOF
[07:00:09][Step 2/2] I170602 06:56:51.296881 16551 vendor/google.golang.org/grpc/server.go:809  grpc: Server.processUnaryRPC failed to write status connection error: desc = "transport: tls: use of closed connection"
[07:00:09][Step 2/2] I170602 06:56:51.297141 13706 kv/transport_race.go:71  transport race promotion: ran 38 iterations on up to 1091 requests
[07:00:09][Step 2/2] E170602 06:56:51.301353 16081 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): failed to send RPC: sending to all 3 replicas failed; last error: rpc error: code = FailedPrecondition desc = grpc: the client connection is closing
[07:00:09][Step 2/2] E170602 06:56:51.302053 16178 ccl/sqlccl/backup.go:428  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while updating progress on job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235'): failed to send RPC: sending to all 3 replicas failed; last error: rpc error: code = FailedPrecondition desc = grpc: the client connection is closing
[07:00:09][Step 2/2] E170602 06:56:51.303381 15358 ccl/sqlccl/backup.go:547  [client=127.0.0.1:33962,user=root,n1] BACKUP ignoring error while marking job 842430060152 (BACKUP DATABASE bench TO 'nodelocal:///go/src/github.com/cockroachdb/cockroach/artifacts/TestBackupRestoreSystemJobs047253235') as successful: node unavailable; try another peer
[07:00:09][Step 2/2] E170602 06:56:51.304325 15358 sql/pgwire/v3.go:205  [n1,client=127.0.0.1:33962] write tcp 127.0.0.1:58952->127.0.0.1:33962: use of closed network connection
[07:00:09][Step 2/2] --- FAIL: TestBackupRestoreSystemJobs (17.94s)
[07:00:09][Step 2/2] 	backup_test.go:465: expected progress to be in range [0.25, 0.75] after 1s but got 0.083333
```

It seems like something has called `stopper.Quiesce()`, but I can't pin down what.

--- a/pkg/ccl/sqlccl/backup_test.go
+++ b/pkg/ccl/sqlccl/backup_test.go
@@ -25,7 +25,6 @@ import (
 	"strings"
 	"sync/atomic"
 	"testing"
-	"time"
 
 	"github.com/kr/pretty"
 	"github.com/lib/pq"
@@ -432,7 +431,7 @@ func verifySystemJobProgress(
 	}
 
 	// Ensure the fractionCompleted of the latest job is in the range [0.25, 0.75].
-	err := util.RetryForDuration(time.Second, func() error {
+	err := util.RetryForDuration(testutils.DefaultSucceedsSoonDuration, func() error {
 		var fractionCompleted float32
 		sqlDB.QueryRow(
 			`SELECT fraction_completed FROM crdb_internal.jobs ORDER BY created DESC LIMIT 1`,

